sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -240,31 +221,45 @@ export interface QueryLogsResult {
     visualization?: any;
 }
 
-// @public
-export interface QueryMetricsOptions extends OperationOptions {
-    aggregations?: string[];
-    filter?: string;
-    interval?: string;
-    metricNames?: string[];
-    metricNamespace?: string;
-    orderBy?: string;
-    resultType?: ResultType;
-    top?: number;
-}
-
 // @public
 export interface QueryMetricsResult {
     cost?: number;
-    interval?: string;
+    granularity?: string;
     metrics: Metric[];
     namespace?: string;
     resourceRegion?: string;
     timespan: string;
```

Comment:
timespan is a string in the output but a TimeInterval in the input side - should we map it back to TimeInterval if we can?